Properties/launchSettings.json

```diff
@@ -1,26 +1,10 @@
 ﻿{
   "$schema": "http://json.schemastore.org/launchsettings.json",
-  "iisSettings": {
-    "windowsAuthentication": false,
-    "anonymousAuthentication": true,
-    "iisExpress": {
-      "applicationUrl": "http://localhost:17855",
-      "sslPort": 44389
-    }
-  },
   "profiles": {
-    "IIS Express": {
-      "commandName": "IISExpress",
-      "launchBrowser": true,
-      "launchUrl": "weatherforecast",
-      "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
-      }
-    },
     "Explorer": {
       "commandName": "Project",
-      "launchBrowser": true,
-      "launchUrl": "weatherforecast",
+      "launchBrowser": false,
+      "launchUrl": "explore",
```

Comment:
This seems wrong? Or it would require that the developers all have some hostname mapping of `explore` to `localhost`? Or am I misunderstanding what `launchUrl` does?

Reply:
The description for the `launchUrl` field is: "The relative URL to launch in the browser". Since we use `POST` requests, I believe that nobody will use the browser for accessing the endpoint, so  in the end we will not use this feature for anything.

Reply:
ah, thanks for the clarification

Reply:
I investigated a little bit more, and it seems that the launch.json is onyl used by Visual Studio. So considering our choice of tools, it seems that we don't need that file, so I will remove it.

Reply:
And I am thinking to also modify the other settings file, that is actually used by VSCode, `launch.json`, so that it will not automatically launch the browser when we run/debug the project from VSCode.